### DISCUSSION TITLE
How do I use the swaggerUI/OpenAPI?

### DISCUSSION THREAD

**Discussion by jonathanvty** [NUWCDIVNPT/stig-manager#1282] [General]
I am trying to "try it out on" on any of the endpoint located in the SwaggerUI and I keep receiving a 401 permission error. How do I Authorize my requests before sending them?

**Comment by jonathanvty** [comment]
Is there existing documentation for how to used this tool? (SwaggerUI)

**Comment by csmig** [comment]
We serve the standard Swagger UI from the API container, from the endpoint `/api-docs`. It is pre-loaded with the STIG Manager API definition and OIDC server values used by the serving API. Perhaps this short video will help with the authentication. We don't currently provide separate Swagger UI documentation specific to STIG Manager. 

https://github.com/NUWCDIVNPT/stig-manager/assets/33138761/b7e5a6f8-9a9d-453f-af9e-feb288991a13
